packages/wrangler/src/config/index.ts

```diff
@@ -618,13 +618,24 @@ export function printBindings(
 		return;
 	}
 
+	let title: string;
+	if (context.provisioning) {
+		title = "The following bindings need to be provisioned:";
+	} else if (context.name && getFlag("MULTIWORKER")) {
+		title = `${chalk.blue(context.name)} has access to the following bindings:`;
+	} else {
+		title = "Your worker has access to the following bindings:";
+	}
+
 	const message = [
-		`${context.name && getFlag("MULTIWORKER") ? chalk.blue(context.name) : "Your worker"} has access to the following bindings:`,
+		title,
 		...output
 			.map((bindingGroup) => {
 				return [
 					`- ${bindingGroup.name}:`,
-					bindingGroup.entries.map(({ key, value }) => `  - ${key}: ${value}`),
+					bindingGroup.entries.map(
+						({ key, value }) => `  - ${key}${value ? ":" : ""} ${value}`
```

Comment:
this just removes the colon if there is no value (ie for inherited bindings). I do think we maybe should print something for inherited bindings (id? name?) and that's totally possible as we've got the worker settings, but just wanted to discuss first and not block this PR on that.